src/controller/CHIPDeviceController.cpp

```diff
@@ -393,7 +432,8 @@ CHIP_ERROR ChipDeviceController::SendMessage(void * appReqState, PacketBuffer *
         // Try sesion resumption if needed
         if (err != CHIP_NO_ERROR && trySessionResumption)
         {
-            err = ResumeSecureSession();
+            // VerifyOrExit(mRemoteDeviceId.HasValue(), err = CHIP_ERROR_INCORRECT_STATE);
```

Comment:
Why is this commented out?

Reply:
Got commented out during some debugging. Good catch. I'll uncomment it out.